### PR TITLE
Add a new GitHub org which belong to QingCloud

### PR DESCRIPTION
**Pull request by LinuxSuRen**
See also the following screenshots:

image
image

### CODE REVIEW

File: labeled_data/companies/qingcloud/index.yml [hunk 1]
```diff
@@ -9,3 +9,4 @@ data:
     - 86996287 # OpenELB
     - 31385114 # radondb
     - 28995775 # QingCloudAppcenter
+    - 90166053f # quanxiang-cloud
```
Comment (frank-zsy) [inline labeled_data/companies/qingcloud/index.yml:12]: I think you added an extra `f` in the end of the id

**Review by frank-zsy — CHANGES_REQUESTED** [review]

File: labeled_data/companies/qingcloud/index.yml (hunk 1 above)
Comment (LinuxSuRen) [reply labeled_data/companies/qingcloud/index.yml:12]: Good catch!

**Comment by frank-zsy** [comment]
Thanks for the data enhancement

/approve

**Pull request merged**